src/main/resources/graphql/task.graphqls

```diff
@@ -1,4 +1,4 @@
-scalar LocalDateTime
```

Comment:
This shouldn't be removed. LocalDateTime is not included by default as a type in graphql and must be defined in the schema as a custom scalar.

Reply:
When I have it in both  the team.graphql and  the task.graphql , I get this error: LocalDateTime' type tried to redefine existing 'LocalDateTime' type